packages/web-components/fast-foundation/src/toolbar/toolbar.template.ts

```diff
@@ -29,14 +28,30 @@ export const toolbarTemplate: (
     >
         <slot name="label"></slot>
         <div class="positioning-region" part="positioning-region">
-            ${startTemplate}
+            <span class="start start__hidden" part="start" ${ref("startContainer")}>
+                <slot
+                    name="start"
+                    ${slotted({
+                        filter: elements(),
+                        property: "startSlottedItems",
+                    })}
```

Comment:
It seems like this could be generally useful to add across all instances - is there a reason we couldn't do that?

Reply:
True, I'll take a look at doing this, and the implications of it. Should be fine since components using start and end should also extend the StartEnd class.

Reply:
So if we add this we can't really use it, can we? I added this to the StartEnd and tried to access it in the Toolbar class, but it shows the startSlottedItems as empty. I believe this is expected because it is a mixin and not a class extension.

I originally added this to add the __hidden classes based on if items were slotted, but that's not necessary, so I will likely remove this and anything related to my additions for __hidden functionality.

Reply:
Ok, after further investigation having the ${slotted...} directive in the StartEnd template results in there being nothing in the startSlottedItems *inside* the Toolbar class AND inside the StartEnd class? 
Is this because StartEnd is a mixin, not an extended class?

this.start.assignedNodes() shows items just fine in the Toolbar class.

Reply:
I am going to remove this for now since it was just for the __hidden and I can use this.start.assignedNodes() to get the nodes for proper keyboard navigation.

Reply:
I am trying to repro this issue in a stackblitz, but am having issues. Tracking it on my todo list.